### ISSUE TITLE
[BUG] "New for WCAG 2.1" bubble isn't readable

### ISSUE THREAD

**Issue by DaveTryon** [microsoft/accessibility-insights-windows#914]
Please check whether the bug has [already been filed](https://github.com/Microsoft/accessibility-insights-windows/issues).

**Describe the bug**
The "New for WCAG 2.1" label in the Color Contrast analyzer fails color contrast

**To Reproduce**
1. Start AIWin
2. Click on the color contrast button
3. Look at the section labeled "Graphical objects and UI components"

**Expected behavior**
There is supposed to be a bubble saying "New for WCAG 2.1"

**Actual behavior**
The bubble is there, but it isn't readable because the bubble has the wrong background color (see screenshot)

**Screenshots or .GIF**
![image](https://user-images.githubusercontent.com/45672944/91909254-b4f29100-ec61-11ea-8134-28946375b4dd.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 2004
 - Accessibility Insights for Windows Version: 1.1.1339.1

**Additional context**
This looks like a regression related to the color palette work. The fix will need to be validated in all color modes, including all HC modes

**Comment by DaveTryon** [comment]
Resolved fixed in 1.1.1347.1
